### PR TITLE
Generates some new models from the openapi specs

### PR DESCRIPTION
**Pull request by cloudify**
NotificationChannelStatus and NotificationStatus

### CODE REVIEW

File: api/public_api_v1.yaml [hunk 1]
```diff
@@ -210,14 +209,22 @@ definitions:
         $ref: '#/definitions/TimeToLive'
       content:
         $ref: '#/definitions/MessageContent'
-      status:
-        $ref: '#/definitions/MessageStatus'
+      senderOrganizationId:
```
Comment (gunzip) [inline api/public_api_v1.yaml]: why camel cased ?

File: api/public_api_v1.yaml [hunk 2]
```diff
@@ -210,14 +209,22 @@ definitions:
...
       - id
       - fiscal_code
-      - time_to_live
       - content
-      - status
+      - senderOrganizationId
+  MessageStatus:
+    type: object
+    properties:
+      message:
```
Comment (gunzip) [inline api/public_api_v1.yaml]: why message status contains the whole message ?

File: lib/controllers/messages.ts [hunk 3]
```diff
@@ -127,6 +113,22 @@ export const MessagePayloadMiddleware: IRequestMiddleware<
...
+function retrievedMessageToPublic(
+  retrievedMessage: IRetrievedMessage
+): CreatedMessage {
+  return {
+    content: {
+      body_short: retrievedMessage.bodyShort
+    },
+    fiscal_code: retrievedMessage.fiscalCode,
+    id: retrievedMessage.id,
+    senderOrganizationId: retrievedMessage.senderOrganizationId
```
Comment (gunzip) [inline lib/controllers/messages.ts]: sender_organization_id

File: api/public_api_v1.yaml (hunk 1 above)
Comment (cloudify) [reply api/public_api_v1.yaml]: ops leftover

File: api/public_api_v1.yaml (hunk 2 above)
Comment (cloudify) [reply api/public_api_v1.yaml]: better rename it, this is actually the response to `GetMessage`

**Pull request merged**